tests/unit/Endpoints/Webhooks/UpdateOrderTest.php

```diff
@@ -91,35 +91,46 @@ public function dataProviderOrderRefund(): array
         $data = [];
 
         $data['backoffice order without refund'] = [
-          'paidBackValueWt' => 0,
-          'refundedPriceWt' => 50.00,
-          'firstExpectedRefundCount' => 0,
-          'secondExpectedRefundCount' => 1,
+            'paidValueWt' => 50.00,
+            'paidBackValueWt' => 0,
+            'refundedPriceWt' => 50.00,
+            'firstExpectedRefundCount' => 0,
+            'secondExpectedRefundCount' => 1,
         ];
 
         $data['backoffice order with refund'] = [
+            'paidValueWt' => 50.00,
             'paidBackValueWt' => 50.00,
             'refundedPriceWt' => 50.00,
             'firstExpectedRefundCount' => 0,
             'secondExpectedRefundCount' => 0,
         ];
 
+        $data['backoffice order with partial refund'] = [
+            'paidValueWt' => 50.00,
+            'paidBackValueWt' => 25.00,
+            'refundedPriceWt' => 25.00,
```

Comment:
```suggestion
            'paidBackValueWt' => 35.00,
            'refundedPriceWt' => 35.00,
```

makes better test to now split it in half